### PR TITLE
Enhancement: Support for s390x and ppc64le

### LINKED ISSUE #45: Add support for s390x and ppc64le
At the moment targets linux/s390x and linux/ppc64le are not supported.

They are missing in
- build.sh 
- in the dockerFile which doesn't pull the s390x and ppc64le cross compilers.

Should be pretty straightforward. I'll do a PR if needed.

### PR DESCRIPTION
**Pull request by ivan-w**
Please review. Thank you,
--Ivan

Closes #45

### CODE REVIEW

File: docker/base/build.sh [hunk 1]
```diff
@@ -313,6 +313,34 @@ for TARGET in $TARGETS; do
       CC=mips-linux-gnu-gcc-6 CXX=mips-linux-gnu-g++-6 GOOS=linux GOARCH=mips CGO_ENABLED=1 go build $V $X $TP $MOD "${T[@]}" --ldflags="$V $LD" $BM -o "/build/$NAME-linux-mips`extension linux`" $PACK_RELPATH
     fi
   fi
+  if ([ $XGOOS == "." ] || [ $XGOOS == "linux" ]) && ([ $XGOARCH == "." ] || [ $XGOARCH == "s390x" ]); then
+    if [ "$GO_VERSION" -lt 180 ]; then
```
Comment (kolaente) [inline docker/base/build.sh]: This check may not even be needed, since we dropped support for Go < 1.10. It doesn't hurt though.

Comment (ivan-w) [reply docker/base/build.sh]: I'm not sure which version of go added support for s390x and powerpc64le cross compile but apparently it already worked in 1.11 (sorry fairly new to this)

Comment (kolaente) [reply docker/base/build.sh]: Looks like it has support for it since 1.7: https://golang.org/doc/go1.7

Comment (kolaente) [reply docker/base/build.sh]: Same for ppc64le

Comment (ivan-w) [reply docker/base/build.sh]: Do you need a new PR or can it be fixed a after (-lt 170). ? I just want to know if the change is sane :P

Comment (kolaente) [reply docker/base/build.sh]: No need to create a new pr, could you just change that to `-lt 170`? (With a new commit)

Comment (ivan-w) [reply docker/base/build.sh]: Just did - does that look right ?

**Comment by kolaente** [comment]
I'll just wait for the CI to pass, then we can merge this

**Comment by kolaente** [comment]
@ivan-w Please keep me up to date here

**Comment by ivan-w** [comment]
Will do - ppc64le is probably broken (s390x should work fine).
I'll fork back from master ;) (to avoid unnecessary merge commits)

**Comment by kolaente** [comment]
You can continue working here, we usually squash PRs, meaning there will be only one merge commit in master once this is merged.

**Comment by ivan-w** [comment]
I hear you !

**Comment by ivan-w** [comment]
I was a bit worried that the ppc64le doesn''t have specific packages against ppc64 Big Endian - but ubuntu may have both endianness for the libc6 and libc6-dev packaged together. If it passed CI I think we are good to go. If something shows up weird, I'll just do a new PR with what was broken and it can be fixed ;)

**Pull request merged**